core/language/en-GB/Buttons.multids

```diff
@@ -32,6 +32,8 @@ ExportTiddler/Caption: export tiddler
 ExportTiddler/Hint: Export tiddler
 ExportTiddlers/Caption: export tiddlers
 ExportTiddlers/Hint: Export tiddlers
+ExportTWCore/Caption: export tiddlywiki core
```

Comment:
"~TiddlyWiki" should be capitalised, with the tilde character to prevent camelcase linking

Reply:
The wording "TWCore" is awkward, I think it would better to spell it out as "ExportTiddlyWikiCore/Caption" etc.